FPADF_Announcement.md

```diff
@@ -0,0 +1,20 @@
+## <img src="images/freecad.svg" style="zoom:50%;" /> FreeCAD Project Association Development Fund
+
+
+#### Announcement - FPA Development Fund
+
+The FreeCAD Project Association is pleased to announce the creation of the FPA Development Fund (FPADF). The FPADF offers grants to individuals for contributions to FreeCAD.
+
+Initially, grants from the FPADF will be capped at US$1000 and grants will be awarded only for work on the FreeCAD code base.
```

Comment:
```suggestion
Grants are capped at US$1000 and will be awarded only for work on the FreeCAD code base.
```

Reply:
Avoid casting the current announcment as suboptimal or inferior to a future goal.